core/include/gnuradio-4.0/plugin_loader.hpp

```diff
@@ -178,18 +178,17 @@ class plugin_loader {
     }
 
     std::unique_ptr<gr::BlockModel>
-    instantiate(std::string name, std::string_view type, const property_map &params = {}) {
+    instantiate(std::string_view name, std::string_view type, const property_map &params = {}) {
         // Try to create a node from the global registry
         if (auto result = _global_registry->createBlock(name, type, params)) {
             return result;
         }
-
-        auto it = _handler_for_name.find(name);
+        auto it = _handler_for_name.find(std::string(name)); // TODO avoid std::string here
```

Comment:
This could use something like in the answer here: https://stackoverflow.com/questions/34596768/stdunordered-mapfind-using-a-type-different-than-the-key-type not sure what the straight-forward way nowadays is.